### PR TITLE
chore: auto format some TypeScript files

### PR DESCRIPTION
**Pull request by lym953**
### What does this PR do?

Auto format some TypeScript files

### Motivation

The next PR (https://github.com/DataDog/datadog-cdk-constructs/pull/285) will touch lots of files, and saving those files on VS Code on my laptop will auto-format those files, mixing formatting changes and other changes in the same PR, so I want to separate out the formatting changes to this PR to make changes easier to review.

I can also disable "format on save" on my IDE, but I think it's a good thing to keep code in good format.

### Testing Guidelines

`npx projen build`, no error

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [x] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

**Comment by duncanista** [comment]
Isn't it possible to format them with a plugin? Instead of relying on a formatter that is only installed in your computer?

**Comment by lym953** [comment]
@duncanista Right now the format check only covers `src/`. Let me make it cover `.ts` files in `integration_tests/` and `examples/` as well.

See the format check failure on the demo commit: https://github.com/DataDog/datadog-cdk-constructs/actions/runs/10605642445/job/29394854056

File: .projenrc.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     "CONTRIBUTING.md",
   ],
   scripts: {
-    "check-formatting": "prettier --check src/**",
+    "check-formatting": "prettier --check src/** integration_tests/**/*.ts examples/**/*.ts",
```
Comment (lym953) [inline .projenrc.js:70]: I tried to find a more succinct way to get all `.ts` files in the repo, but couldn't find one. For example, I tried `**/*.ts` and `*/**/*.ts`, but both fail to find some files in `integration_tests` or in `examples`.

**Review by duncanista — APPROVED** [review]

**Comment by duncanista** [comment]
@lym953 ok, thanks for the detailed explanation and the comment on what work and what didn't, approving, feel free to take the right decision!

**Pull request merged**